libraries/Microsoft.Bot.Builder.LanguageGeneration/EvaluationOptions.cs

```diff
@@ -43,6 +43,7 @@ public EvaluationOptions()
             NullSubstitution = null;
             LineBreakStyle = null;
             Locale = null;
+            Random = null;
```

Comment:
This is an anti-pattern.  Either assign to each definition or have this, but both is weird.  For example you assign random in the get/set below to a random object, but here you set it to null.  If the default is null you don't need to set anything and if the default is a random object it should be that.